client/signup/config/steps-pure.js

```diff
@@ -654,7 +654,7 @@ export function generateSteps( {
 			stepName: 'launch',
 			apiRequestFunction: launchSiteApi,
 			dependencies: [ 'siteSlug' ],
-			providesDependencies: [ 'isPreLaunch' ],
+			providesDependencies: [],
```

Comment:
I think that `providesDependencies` prop can be removed as well, at least by looking at the config of other steps.

Reply:
Good - guess I went by the opposite observation here without further ado :D